dev/Scroller/Scroller.idl

```diff
@@ -8,26 +8,32 @@ enum InteractionState
     Animation = 3,
 };
 
+#ifndef BUILD_WINDOWS
+
 [WUXC_VERSION_PREVIEW]
 [webhosthidden]
-enum ScrollerScrollMode
+enum ScrollMode
 {
-    Auto = 0,
-    Enabled = 1,
-    Disabled = 2,
+    Enabled = 0,
+    Disabled = 1,
+#ifdef USE_SCROLLMODE_AUTO
+    Auto = 2,
+#endif
```

Comment:
Just curious, what is the motivation for removing auto ? How do i show the scrollbars only if the content is bigger than the viewport ?

Reply:
The motivation was to give us the option to introduce a property in the future that governs the overpan behavior in general. ScrollMode.Auto is about the ability to overpan or not when the content is smaller than the viewport.
In order to show the scrollbar only when the content is larger than the viewport, use the ScrollBarVisibility.Auto value.